R/tm_g_butterfly.R

```diff
@@ -40,41 +40,33 @@
 #' @template author_liaoc10
 #'
 #' @examples
-#'
 #' # Example using stream (ADaM) dataset
-#' library(dplyr)
-#' library(nestcolor)
+#' data <- teal.data::cdisc_data() |>
+#'   within({
+#'     library(dplyr)
+#'     library(nestcolor)
+#'   }) |>
+#'   within(set.seed(23)) |>
+#'   within(ADSL <- osprey::rADSL) |>
+#'   within(ADAE <- osprey::rADAE) |>
+#'   within(ADSL <- mutate(ADSL, DOSE = paste(sample(1:3, n(), replace = TRUE), "UG"))) |>
+#'   within(ADAE <- mutate(
+#'     ADAE,
+#'     flag1 = ifelse(AETOXGR == 1, 1, 0),
+#'     flag2 = ifelse(AETOXGR == 2, 1, 0),
+#'     flag3 = ifelse(AETOXGR == 3, 1, 0),
+#'     flag1_filt = rep("Y", n())
+#'   ))
```

Comment:
I thought it's more natural to build data one expression at a time and it would be easier to understand for someone who sees `within` for the first time.

Reply:
I have opposite opinion. Each code element mixed in each line with `within` makes `code` hard to read. Let others decide on this